src/formatters/deployResultFormatter.ts

```diff
@@ -155,4 +161,100 @@ export class DeployResultFormatter extends ResultFormatter {
       }
     }
   }
+
+  protected verboseTestFailures(): void {
+    if (this.result?.response?.numberTestErrors) {
+      const failures = normalizeToArray(this.result.response.details?.runTestResult?.failures);
+
+      const tests = this.sortTestResults(failures);
+
+      this.ux.log('');
+      this.ux.styledHeader(
+        chalk.red(`Test Failures [${asString(this.result.response.details.runTestResult?.numFailures)}]`)
+      );
+      this.ux.table(tests, {
+        columns: [
+          { key: 'name', label: 'Name' },
+          { key: 'methodName', label: 'Method' },
+          { key: 'message', label: 'Message' },
+          { key: 'stackTrace', label: 'Stacktrace' },
+        ],
+      });
+    }
+  }
+
+  protected verboseTestSuccess(): void {
+    const success = normalizeToArray(this.result?.response?.details?.runTestResult?.successes);
+    if (success.length) {
+      const tests: Successes[] = this.sortTestResults(success);
+      this.ux.log('');
+      this.ux.styledHeader(chalk.green(`Test Success [${success.length}]`));
+      this.ux.table(tests, {
+        columns: [
+          { key: 'name', label: 'Name' },
+          { key: 'methodName', label: 'Method' },
+        ],
+      });
+    }
+    const codeCoverage = normalizeToArray(this.result?.response?.details?.runTestResult?.codeCoverage);
+
+    if (codeCoverage.length) {
+      const coverage = codeCoverage.sort((a, b) => {
+        return a.name.toUpperCase() > b.name.toUpperCase() ? 1 : -1;
+      });
+
+      this.ux.log('');
+      this.ux.styledHeader(chalk.blue('Apex Code Coverage'));
+      this.ux.table(coverage, {
+        columns: [
+          { key: 'name', label: 'Name' },
+          {
+            key: 'numLocations',
+            label: '% Covered',
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore the format method expects two string params, we're passing in a CodeCoverage object
+            format: (numLocations: string, row: CodeCoverage): string => {
+              const numLocationsNum = parseInt(numLocations, 10);
+              const numLocationsNotCovered: number = parseInt(row.numLocationsNotCovered, 10);
+              const color = numLocationsNotCovered > 0 ? chalk.red : chalk.green;
+
+              let pctCovered = 100;
+              const coverageDecimal: number = parseFloat(
+                ((numLocationsNum - numLocationsNotCovered) / numLocationsNum).toFixed(2)
+              );
+              if (numLocationsNum > 0) {
+                pctCovered = coverageDecimal * 100;
+              }
+
+              return color(`${pctCovered}%`);
+            },
+          },
+          {
+            key: 'locationsNotCovered',
+            label: 'Uncovered Lines',
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore the format method expects a string, we're passing in LocationsNotCovered[]
```

Comment:
What if we build the string separately and set it on `format` so we don't need the `@ts-ignore`